src/ui/ui_settings/__tests__/ui_settings_mixin_integration.js

```diff
@@ -34,13 +34,13 @@ import { uiSettingsMixin } from '../ui_settings_mixin';
 describe('uiSettingsMixin()', () => {
   const sandbox = sinon.createSandbox();
 
-  async function setup(options = {}) {
-    const {
-      enabled = true
-    } = options;
-
+  async function setup() {
     const config = await Config.withDefaultSchema({
-      uiSettings: { enabled }
+      uiSettings: {
+        overrides: {
```

Comment:
Do we still need the `overrides` "namespace"? If we still need a namespace, maybe "controlledByServer" would be more consistent?

Reply:
I'd like to leave the option for custom `defaults:` down the road, and I feel like `controlledByServer` makes more sense from the UI perspective, but less sense in the server config itself.